docs/_docs/getting-started.md

```diff
@@ -153,6 +141,36 @@ $ tributors update zenodo
 $ tributors update codemeta
 ```
 
+If the client finds more than one orcid identifier for a name, you'll be prompted
+to run in `--interactive` mode. Running in interactive mode will allow you
+to choose a number for each one:
+
+```bash
+$ tributors update zenodo --interactive
+INFO:    zenodo:Updating .zenodo.json
+INFO:    zenodo:Updating .tributors cache from .zenodo.json
+
+Meyer, Kyle
+======================================================
+[1]
+  Name: Meyer, Kyle
+  Orcid: 0000-0002-1933-2908
+  Institutions: University of California Davis, University of Michigan, University of Texas at Austin
+
+[2]
+  Name: Meyer, Kyle
+  Orcid: 0000-0001-8632-4425
+  Institutions: Kroger Co, Northeastern Ohio Medical University, University of Toledo, University of Toledo Medical Center
+
+[3]
+  Name: Meyer, Kyle
+  Orcid: 0000-0001-8846-7411
+  Institutions: University of Auckland, University of Oregon, University of Wisconsin Milwaukee
+
+Please enter a choice, or s to skip.
+[1:3 or s to skip] : 
```

Comment:
unfortunately neither of those -- he has none.  If reran later -- would it ask again?

Reply:
Yes.

Reply:
So then there should be an option to skip in the future

Reply:
There is! Take a look at the message at the bottom - s means skip :)

Reply:
And I don’t think it should skip it in the future, because it’s very likely that a record might be added or available that wasn’t before.

Reply:
So on every rerun I would need to skip the same person who never bothered to get orcid?

Reply:
Yes, one click of the s key, a fairly reasonable thing for an interactive client. If it were upwards of 50 times this would be a pain, but it’s probably not more than a handful. And if those small number of times were annoying enough the user would have more incentive to nudge the contributor to create a profile, which is the best outcome for everyone.